l10n_it_fiscalcode/__manifest__.py

```diff
@@ -5,12 +5,15 @@
 
 {
     'name': 'Italian Localisation - Fiscal Code',
-    'version': '10.0.1.0.1',
+    'version': '10.0.1.1.0',
     'category': 'Localisation/Italy',
     'author': "Odoo Italia Network, Odoo Community Association (OCA)",
     'website': 'https://odoo-community.org/',
     'license': 'AGPL-3',
-    'depends': ['base_vat'],
+    'depends': [
+        'base_vat',
+        'web_widget_char_switchcase'
```

Comment:
Could you avoid to add this dependency? Instead, you could add a paragraph in the readme that recommends to use this module in combination with `web_widget_char_switchcase` (not mandatory). This way the `options="{'transform': 'upper'}"` would be effective only when `web_widget_char_switchcase` is installed. In case `web_widget_char_switchcase` is not installed, nothing changes for the end user.

Reply:
Done in https://github.com/OCA/l10n-italy/pull/607/commits/db7b6b1299ce7ca3d5e00c5cd342b70edf957446